Gemfile.lock

```diff
@@ -32,12 +32,12 @@ PLATFORMS
   ruby
 
 DEPENDENCIES
-  bundler (~> 1.17)
+  bundler
   fmparser!
-  google-protobuf (~> 3.8)
+  google-protobuf
   pry (~> 0.12)
   rake (~> 10.0)
   rspec (~> 3.8)
 
 BUNDLED WITH
-   1.17.2
+   2.4.2
```

Comment:
(note)
- latest bunlder version and it require => 2.6 ruby version. 
https://rubygems.org/gems/bundler/versions/2.4.2
- bundler 2.3+ lock the version that configured in Gemfile.lock and install missing version by itself
  - https://bundler.io/blog/2022/01/23/bundler-v2-3.html